apps/masterbots.ai/components/routes/browse/browse-category-tabs.tsx

```
@@ -2,83 +2,80 @@
+	return (
+		<div className="w-full py-[10px] my-3 !overflow-x-auto overflow-y-hidden whitespace-nowrap scrollbar small-thumb">
+			<BrowseCategoryButton
+				id="browse-category-tab__null"
+				onClick={() => setActiveTab(null)}
+				category="all"
+				activeTab={activeTab}
+			/>
+			{categories.map((category, key) => (
+				<BrowseCategoryButton
+					id={`browse-category-tab__${category.categoryId}`}
+					key={key}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)